plone/app/upgrade/__init__.py

```diff
@@ -1,19 +1,3 @@
-from plone.app.upgrade.utils import alias_module
-
-import pkg_resources
-
-
-try:
-    pkg_resources.get_distribution("Products.CMFQuickInstallerTool")
-except pkg_resources.DistributionNotFound:
-    # The alias module helps when migrating to Plone 6.0.0a1.
-    # Remove when we no longer support upgrading from Plone 5.2.
-    from . import bbb_qi
-
-    alias_module("Products.CMFQuickInstallerTool", bbb_qi)
-    alias_module("Products.CMFPlone.QuickInstallerTool", bbb_qi)
```

Comment:
I would rather not drop support for upgrading from Plone 5 until Plone 7.

Reply:
Indeed this should stay.

Reply:
No problem, I revert that and update it to `importlib.metadata` 👍🏾